test/asset/api/old/EnumWithCasesBeingChanged.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace RoaveTestAsset;
+
+enum EnumWithCasesBeingChanged
+{
+    case March;
+    case April;
+    case May;
+    case June;
+    case July;
+    case August;
+    case September;
+    case October;
+    case November;
+    Case December;
```

Comment:
What about CasEsensiTIVitY issues?

Reply:
Case names are case sensitive, but I'm not sure what issue you see with that.  I'm using `\Roave\BetterReflection\Reflection\ReflectionEnum::hasCase` which is use the case sensitive `array_key_exists` . I can't see any code that means a combination of cases with different names that are equal under case-insensitive comparison needs any different handling to any other combination of cases with different names.

Reply:
@bdsl I was thinking about renaming `December` to `december` (as an example) - does that result in a BC issue?

Reply:
Right. I'll test it. I expect it will be reported as two BC issues, removal of `December` and addition of `december`. That seems like correct behaviour.

Reply:
It's just about adding that difference to this existing test, BTW :+1:

Reply:
Added a change of case to the test.